test/testdata/rewriter/struct.rb.rewrite-tree.exp

```diff
@@ -13,29 +13,45 @@ class <emptyTree><<C <root>>> < (::<todo sym>)
   end
 
   class <emptyTree>::<C RealStruct><<C <todo sym>>> < (::<todo sym>)
-    class <emptyTree>::<C A><<C <todo sym>>> < (::<root>::<C Struct>)
+    class <emptyTree>::<C A$1><<C <todo sym>>> < (::<root>::<C Struct>)
+      ::Sorbet::Private::Static.sig(::T::Sig::WithoutRuntime) do ||
+        <self>.returns(::T.untyped())
+      end
+
       def foo<<todo method>>(&<blk>)
-        ::T.unsafe(::Kernel).raise("Sorbet rewriter pass partially unimplemented")
+        ::Kernel.raise("Sorbet rewriter pass partially unimplemented")
+      end
+
+      ::Sorbet::Private::Static.sig(::T::Sig::WithoutRuntime) do ||
+        <self>.params(:foo, ::T.untyped()).returns(::T.untyped())
       end
 
       def foo=<<todo method>>(foo, &<blk>)
-        ::T.unsafe(::Kernel).raise("Sorbet rewriter pass partially unimplemented")
+        ::Kernel.raise("Sorbet rewriter pass partially unimplemented")
+      end
+
+      ::Sorbet::Private::Static.sig(::T::Sig::WithoutRuntime) do ||
+        <self>.returns(::T.untyped())
       end
 
       def bar<<todo method>>(&<blk>)
-        ::T.unsafe(::Kernel).raise("Sorbet rewriter pass partially unimplemented")
+        ::Kernel.raise("Sorbet rewriter pass partially unimplemented")
+      end
+
+      ::Sorbet::Private::Static.sig(::T::Sig::WithoutRuntime) do ||
+        <self>.params(:bar, ::T.untyped()).returns(::T.untyped())
       end
 
       def bar=<<todo method>>(bar, &<blk>)
-        ::T.unsafe(::Kernel).raise("Sorbet rewriter pass partially unimplemented")
+        ::Kernel.raise("Sorbet rewriter pass partially unimplemented")
       end
 
       ::Sorbet::Private::Static.sig(::T::Sig::WithoutRuntime) do ||
         <self>.params(:foo, ::BasicObject, :bar, ::BasicObject).void()
       end
 
       def initialize<<todo method>>(foo = nil, bar = nil, &<blk>)
-        ::T.unsafe(::Kernel).raise("Sorbet rewriter pass partially unimplemented")
+        ::Kernel.raise("Sorbet rewriter pass partially unimplemented")
```

Comment:
This makes it easier to use `Struct.new` in a `# typed: strong` file (though it'll still be imperfect, as the call to `super` will look like a usage of `T.untyped`.